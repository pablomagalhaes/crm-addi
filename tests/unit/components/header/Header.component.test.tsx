import React from 'react';
import { shallow } from 'enzyme';

import Header from '@/components/header/Header.component';

describe('<Header />', () => {
  it('should match snapchat', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
