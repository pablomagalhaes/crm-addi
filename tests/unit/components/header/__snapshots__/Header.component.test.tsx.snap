// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should match snapchat 1`] = `
<styled.div>
  <styled.div>
    <styled.div>
      <Styled(Link)
        to="/"
      >
        <AddiLogo />
      </Styled(Link)>
      <Styled(FontAwesomeIcon) />
    </styled.div>
  </styled.div>
</styled.div>
`;
